astro/src/content/docs/release-notes/index.mdx

```diff
@@ -84,6 +85,20 @@ Update dependencies.
   See the OAuth [Scopes](/docs/lifecycle/authenticate-users/oauth/scopes) documentation for more information, in particular the `Relationship`, `Unknown scope policy`, and `Scope handling policy` configurations.
 </GeneralMigrationWarning>
 
+<GeneralMigrationWarning>
+  If you are using IFRAMEs to access the FusionAuth hosted login pages please check that the IFRAME `src` is from the same domain as the FusionAuth pages.
+
+  [FusionAuth uses cookies](/docs/reference/cookies) to manage user state with the `SameSite` attribute set to `Lax` or `Strict`. Browsers will block `Set-Cookie` headers on cross-domain requests.
+
+  This release introduces a new redirect into the OAuth flows to `/oauth2/consent` as part of the OAuth [Scopes](/docs/lifecycle/authenticate-users/oauth/scopes)
+  feature. This redirect will occur during each browser based interactive OAuth workflow. Prior to this version it was possible to complete an OAuth code grant
+  flow without cookies being set as long as there were no additional redirects to FusionAuth before the final redirect to the configured
+  <InlineField>redirect_url</InlineField>. As a result it did not matter if the `Set-Cookie` headers were blocked.
```

Comment:
As a result it did not matter if the `Set-Cookie` headers were blocked.  In other words, previously, it was possible to have a technically non-working integration (Set-Cookie fails) that still allowed an `auth code` to be returned from an <iframe> call.